### PR TITLE
whole bunch of changes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
The reason why I changed the position of title text is because it looked off center and I have self-diagnosed OCD (self-diagnosed as in i just dont like things that look off) because I added the version number. (just explaining in case someone asks why)

**Comment by Peter0x44** [comment]
Can you keep it as one PR per change please? Also, it would be good if you used `git pull --rebase` so there aren't so many merge commits

**Comment by ghost** [comment]
Yeah, I had just forgotten to make a new branch for these changes and I had another PR... :| 
also yeah I'll rebase next time.

**Comment by ghost** [comment]
well I don't feel like closing this PR because I committed all of this on master

File: src/mainmenu.h [hunk 1]
```diff
@@ -29,6 +29,8 @@ class MainMenu: public GameState
 	Vector2 StartDims;
 	int TitleFontSize;
 
+	Vector2 VersionPos;
```
Comment (Peter0x44) [inline src/mainmenu.h]: Are these variables actually used? The version number text should just be part of the title screen

File: src/mainmenu.cpp [hunk 2]
```diff
@@ -6,9 +6,10 @@ void MainMenu::logic(void)
 	//Determine size and position of Title Text
 	TitleFontSize = GetScreenHeight()/10;
 	TitleDims = MeasureTextEx(GetFontDefault(), "Geometry Slash", TitleFontSize, TitleFontSize/10);
-	TitlePos.x = GetScreenWidth()/2 - TitleDims.x/2;
+	TitlePos.x = GetScreenWidth()/2 - TitleDims.x/1.6;
```
Comment (Peter0x44) [inline src/mainmenu.cpp]: I think the proper change is to add the version number to the MeasureTextEx call

File: src/globals.h [hunk 3]
```diff
@@ -1,5 +1,6 @@
 #ifndef GLOBALS_H
 #define GLOBALS_H
+#define VERSION_NUMBER "1.0b"
```
Comment (Peter0x44) [inline src/globals.h]: I would move this down in the header, there's no reason for it to be on top

File: src/mainmenu.h (hunk 1 above)
Comment (ghost) [reply src/mainmenu.h]: forgot to remove that

**Comment by ghost** [comment]
dammit git

**Comment by Peter0x44** [comment]
I took the basic changes and added them myself - I would like to have more discussion about the other changes to the colors

**Pull request closed without merging**